lib/gui/components/safe-webview/safe-webview.js

```diff
@@ -84,6 +84,14 @@ class SafeWebview extends react.PureComponent {
       this.refs.webview.addEventListener('new-window', this.constructor.newWindow);
       this.refs.webview.addEventListener('console-message', this.constructor.consoleMessage);
 
+      if (this.props.onStateChange) {
+        this.props.onStateChange((event, toState, toParams, fromState) => {
+          if (fromState.name === 'success' && this.state.shouldLoad) {
+            this.refs.webview.src = this.props.src;
```

Comment:
My thought here was in case the page has been updated, I assume some people leave Etcher open constantly to burn many cards, so this would make them see updated pages.